### PR TITLE
add flink-connector catalog

### PR DESCRIPTION
**Pull request by Nicole00**
support nebula schema operations in flink

### CODE REVIEW

File: tools/nebula-flink/src/main/java/org.apache.flink/connector/nebula/catalog/AbstractNebulaCatalog.java [hunk 1]
```diff
@@ -6,20 +6,248 @@
...
+        this.username = username;
+        this.password = pwd;
+        this.address = address;
+    }
+
+
+
+    @Override
+    public void open() throws CatalogException{
+        // test graphClient connection
```
Comment (darionyaphet) [inline tools/nebula-flink/src/main/java/org.apache.flink/connector/nebula/catalog/AbstractNebulaCatalog.java:54]: test  meta client ? why?

Comment (Nicole00) [reply tools/nebula-flink/src/main/java/org.apache.flink/connector/nebula/catalog/AbstractNebulaCatalog.java:54]: fail early if we cannot connect to nebula

Comment (darionyaphet) [reply tools/nebula-flink/src/main/java/org.apache.flink/connector/nebula/catalog/AbstractNebulaCatalog.java:54]: Actually  you create a meta client ...

Comment (Nicole00) [reply tools/nebula-flink/src/main/java/org.apache.flink/connector/nebula/catalog/AbstractNebulaCatalog.java]: modify the comment to "test metaClient connection"

File: tools/nebula-flink/src/main/java/org.apache.flink/connector/nebula/utils/DataTypeEnum.java [hunk 2]
```diff
@@ -23,4 +23,14 @@ public boolean isVertex(){
...
     }
+
+    public static boolean checkValidDataType(String type){
+        if(VERTEX.name().equalsIgnoreCase(type)){
+            return true;
+        }
+        if(EDGE.name().equalsIgnoreCase(type)){
+            return true;
+        }
+        return false;
```
Comment (darionyaphet) [inline tools/nebula-flink/src/main/java/org.apache.flink/connector/nebula/utils/DataTypeEnum.java:34]: merge into one line

**Review by darionyaphet — APPROVED** [review]

**Pull request merged**